user-console/src/main/java/org/pentaho/mantle/client/admin/ChangePasswordDialog.java

```diff
@@ -40,6 +42,8 @@ public class ChangePasswordDialog extends GwtDialog implements ServiceCallback {
   private Button acceptBtn = new Button( Messages.getString( "ok" ) );
   private Button cancelBtn = new Button( Messages.getString( "cancel" ) );
   private boolean acceptBtnEnabled = false;
+  private final String ALLOWED_CHARS = "^[a-zA-Z0-9_.,:;<>|!@#$%^&*()\\[\\]]+$";
+  private final RegExp ALLOWED_CHARS_REGEXP = RegExp.compile( ALLOWED_CHARS );
```

Comment:
Similar to above: check if it can this be static or if it affects the conversion to JavaScript.